repos.json

```diff
@@ -38,7 +38,9 @@
     { "language": "java", "repo": "googleapis/java-asset"},
     { "language": "java", "repo": "googleapis/java-automl"},
     { "language": "java", "repo": "googleapis/java-bigquery"},
+    { "language": "java", "repo": "googleapis/java-bigqueryconnections"},
```

Comment:
Could I trouble you to also add these under the right places in `users.json`?  I know it's annoying you need to do it twice :(

Reply:
Done.